cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShowFunctionsTest.scala

```diff
@@ -28,63 +28,158 @@ class ShowFunctionsTest extends DocumentingTest {
+      login("jake", "abc123")
+      query("SHOW FUNCTIONS EXECUTABLE BY CURRENT USER YIELD *", ResultAssertions(p => {
+        p.columns should contain theSameElementsAs Array("name", "category", "description", "signature", "isBuiltIn", "argumentDescription", "returnDescription", "aggregating", "rolesExecution", "rolesBoostedExecution")
+        p.columnAs[List[String]]("rolesExecution").foreach(_ should be(null))
+        p.columnAs[List[String]]("rolesBoostedExecution").foreach(_ should be(null))
+      })) {
+        limitedResultTable(List("name", "category", "description", "rolesExecution", "rolesBoostedExecution"))
       }
-      section("Listing functions with other filtering") {
-        database("system")
-        initQueries(
-          "CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED"
-        )
-        database("neo4j")
-        p(
-          """The listed functions can also be filtered on whether a user can execute them.
-            #This filtering is only available through the `EXECUTABLE` clause and not through the `WHERE` clause.
-            #This is due to using the users privileges instead of just filtering on the available output columns.""".stripMargin('#'))
-        p(
-          """There are two versions of the `EXECUTABLE` clause.
-            #The first one is to filter for the current user:""".stripMargin('#'))
-        login("jake", "abc123")
-        query("SHOW FUNCTIONS EXECUTABLE BY CURRENT USER YIELD *", ResultAssertions(p => {
-          p.columns should contain theSameElementsAs Array("name", "category", "description", "signature", "isBuiltIn", "argumentDescription", "returnDescription", "aggregating", "rolesExecution", "rolesBoostedExecution")
-          p.columnAs[List[String]]("rolesExecution").foreach(_ should be(null))
-          p.columnAs[List[String]]("rolesBoostedExecution").foreach(_ should be(null))
-        })) {
-          limitedResultTable(List("name", "category", "description", "rolesExecution", "rolesBoostedExecution"))
-        }
-        p("Notice that the two roles columns are empty due to missing the <<administration-security-administration-dbms-privileges-role-management, SHOW ROLE>> privilege.")
-        logout()
-        p("The second versions is to filter for a specific user:")
-        query("SHOW FUNCTIONS EXECUTABLE BY jake", ResultAssertions(p => {
-          p.columns should contain theSameElementsAs Array("name", "category", "description")
-        })) {
-          limitedResultTable(List("name", "category", "description"))
-        }
+      p("Notice that the two roles columns are empty due to missing the <<administration-security-administration-dbms-privileges-role-management,`SHOW ROLE`>> privilege.")
+      logout()
+      p("The second versions is to filter for a specific user:")
+      query("SHOW FUNCTIONS EXECUTABLE BY jake", ResultAssertions(p => {
+        p.columns should contain theSameElementsAs Array("name", "category", "description")
+      })) {
```

Comment:
All descriptions render as code blocks and within quotation marks. It's not strictly wrong, but it doesn't look so nice.